Magento_Swatches/templates/product/layered/renderer.phtml

```diff
@@ -28,61 +30,67 @@
                     $swatchData['swatches'][$option]['value']); ?>
                     <?php $swatchImagePath = $block->getSwatchPath('swatch_image',
                     $swatchData['swatches'][$option]['value']); ?>
-                    <a
-                        href="<?= $label['link'] ?>"
-                        aria-label="<?= $label['label'] ?>"
-                    >
-                        <div
-                            tabindex="-1"
-                            option-type="2"
-                            option-id="<?= $option ?>"
-                            option-label="<?= $label['label'] ?>"
-                            option-tooltip-thumb="<?= $swatchThumbPath ?>"
-                            style="background: url(<?= $swatchImagePath ?>) no-repeat center; background-size: cover;"
-                        ></div>
-                    </a>
+                    <li>
+                        <a
+                            href="<?= $label['link'] ?>"
+                            aria-label="<?= $label['label'] ?>"
+                        >
+                            <div
+                                tabindex="-1"
```

Comment:
I think it's the same case as above, but please verify it with screen reader. If it's color swatch you can add `title` attribute with label value - on hover the name of color will be shown, I think it's good UX option for non-disabled users too ;)

Reply:
Done